keyboards/kingly_keys/ave/staggered/rules.mk

```diff
@@ -0,0 +1 @@
+LAYOUTS = staggered staggered_big_boy
```

Comment:
Not valid community layouts, so should be removed. Blank file is fine, as it need it to become a make target
```suggestion
```